bson/src/test/unit/org/bson/codecs/jsr310/InstantCodecSpecification.groovy

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bson.codecs.jsr310
+
+import org.bson.BsonDocument
+import org.bson.codecs.Codec
+import org.bson.codecs.configuration.CodecConfigurationException
+import spock.lang.IgnoreIf
+
+@IgnoreIf({ javaVersion < 1.8 })
+class InstantCodecSpecification extends JsrSpecification {
+
+    def 'should round trip Instant successfully'() {
+        when:
+        def writer = encode(instant)
+
+        then:
+        writer.getDocument().get('key').asDateTime().value == millis
+
+        when:
+        java.time.Instant actual = decode(writer)
+
+        then:
+        instant == actual
+
+        where:
+        instant                                                                             | millis
+        java.time.Instant.EPOCH                                                             | 0
```

Comment:
Why all the qualified references in the specification tests?  Can they be replaced with imports?

Reply:
I thought that it would fail on java < 8 but evergreen has shown thats not the case.